src/core/01_network_pair.tf

```diff
@@ -0,0 +1,36 @@
+resource "azurerm_resource_group" "rg_pair_vnet" {
+  name     = format("${local.project_pair}-vnet-rg", )
+  location = var.location_pair
+
+  tags = var.tags
+}
+
+module "vnet_pair" {
+  source               = "git::https://github.com/pagopa/azurerm.git//virtual_network?ref=v4.15.0"
+  name                 = "${local.project_pair}-vnet"
+  location             = azurerm_resource_group.rg_pair_vnet.location
+  resource_group_name  = azurerm_resource_group.rg_pair_vnet.name
+  address_space        = var.cidr_pair_vnet
+  ddos_protection_plan = var.ddos_protection_plan
+
+  tags = var.tags
+}
+
+## Peering between the vnet(main) and integration vnet
+module "vnet_peering_pair_vs_core" {
+  source = "git::https://github.com/pagopa/azurerm.git//virtual_network_peering?ref=v4.15.0"
+
+  location = azurerm_resource_group.rg_vnet.location
+
+  source_resource_group_name       = azurerm_resource_group.rg_pair_vnet.name
+  source_virtual_network_name      = module.vnet_pair.name
+  source_remote_virtual_network_id = module.vnet_pair.id
+  source_allow_gateway_transit     = false
```

Comment:
please add the parameter `source_allow_forwarded_traffic` and `target_allow_forwarded_traffic` to all new peerings